core/deployment/src/test/java/io/quarkiverse/operatorsdk/test/sources/ConfiguredReconciler.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkiverse.operatorsdk.test.sources;
+
+import io.javaoperatorsdk.operator.api.config.ControllerConfigurationOverrider;
+import io.javaoperatorsdk.operator.api.reconciler.Context;
+import io.javaoperatorsdk.operator.api.reconciler.Reconciler;
+import io.javaoperatorsdk.operator.api.reconciler.UpdateControl;
+import io.quarkiverse.operatorsdk.runtime.api.ConfigurableReconciler;
+
+public class ConfiguredReconciler implements Reconciler<TestCR>, ConfigurableReconciler<TestCR> {
```

Comment:
Just occured to me, why ConfigurableReconciler doesn't extend Recociler?

Reply:
Should be done now.